taxi_service/settings.py

```diff
@@ -133,3 +133,6 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+
+LOGIN_REDIRECT_URL = "http://localhost:8000"
```

Comment:
`LOGIN_REDIRECT_URL` should not contains domain, because your site can have difference domains
```suggestion
LOGIN_REDIRECT_URL = "/"
```

Reply:
Thank`s ❤️